routes/donations.js

```diff
@@ -0,0 +1,168 @@
+const express = require('express');
+
+const { pool, db } = require('../server/db');
+
+const router = express.Router();
+
+// get all donation rows
+router.get('/', async (req, res) => {
+  try {
+    const allDonations = await pool.query(`SELECT * FROM Donations`);
+    res.status(200).json(allDonations.rows);
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// get specific donation
+router.get('/:donationId', async (req, res) => {
+  try {
+    const { donationId } = req.params;
+    const donation = await pool.query(`SELECT * FROM Donations WHERE id = ${donationId}`);
```

Comment:
generally for simple queries where variables are passed in, this syntax is better for avoiding sql injection

```JavaScript
await pool.query(`SELECT * from donations WHERE id = $1`, [routeId])
```
![image](https://user-images.githubusercontent.com/45449494/201872742-119ff22b-2ee8-4fb6-bb33-f9fa46121a7f.png)

Reply:
see this [article](https://www.w3schools.com/sql/sql_injection.asp) for examples of why this should be avoided.